app/src/test/java/uk/ac/ebi/atlas/experimentpage/TSnePlotSettingsServiceIT.java

```diff
@@ -162,7 +161,9 @@ private long getOpenFileCount() {
         }
     }
 
-    private Stream<String> randomSingleCellExperimentAccessionProvider() {
-        return Stream.of(jdbcTestUtils.fetchRandomExperimentAccession());
+    private Stream<Arguments> randomSingleCellExperimentAccessionProvider() {
```

Comment:
The method name is not reflecting what the method is doing. It is not only returning a random experiment accession, but also a plot method.
Please, either extract that other functionality to another helper method or change the name of the method.
Thanks